### DISCUSSION TITLE
Export to CSV results in a blank file

### DISCUSSION THREAD

**Discussion by fraize** [RafaelGB/obsidian-db-folder#530] [Q&A] [answered]
I've created a simple DB Folder with four rows of basic text columns, nothing fancy. Selecting the hamburger menu to "Export CSV" results in a "new database.csv" file with nothing in it. Is this feature yet-to-be-implemented?
![image](https://user-images.githubusercontent.com/3311472/196726558-0fcb6f96-bfd7-4073-9e07-e761a2d85723.png)

**Comment by RafaelGB** [comment, marked as answer]
Hi @fraize,
Could you attach a zip with the notes? I will open an issue with it. It should work without problems

**Reply by fraize** [reply]
Thank you for having a look at this.
[db Folder - no CSV from Export.zip](https://github.com/RafaelGB/obsidian-db-folder/files/9824678/db.Folder.-.no.CSV.from.Export.zip)

**Reply by RafaelGB** [reply]
This will be fixed with the next version

https://github.com/RafaelGB/obsidian-db-folder/issues/535#issuecomment-1285360675

Thanks for your report!

**Reply by RafaelGB** [reply]
This bug was fixed with 2.7.4

Could you confirm if its resolved?

**Reply by mylenrath** [reply]
I use version 3.2.4 but I have the same issue.  I have several database folders but export to csv does not work for any of them.  Result is a blank file.

**Comment by fraize** [comment]
Confirmed resolved. CSVs are outputting as expected.
